website/docs/index.html.markdown

```diff
@@ -105,3 +110,55 @@ The following arguments are supported:
   Version](https://www.postgresql.org/support/versioning/) or `current`.  Once a
   connection has been established, Terraform will fingerprint the actual
   version.  Default: `9.0.0`.
+
+## GoCloud
+
+By default, the provider uses [lib/pq][libpq] library to connect to PostgreSQL servers, but you can switch to [GoCloud](https://gocloud.dev/howto/sql/).
+GoCloud simplifies the connection to AWS/GCP hosted databases. For example GCP requires a custom proxy with authentication credentials, GoCloud manages it automatically.
+
+### AWS
+
+To enable GoCloud for AWS RDS, set `scheme` to `awspostgres`, `host` is the database endpoint in RDS.
+(e.g.: `instance.xxxxxx.region.rds.amazonaws.com`)
+
```

Comment:
Although I haven't tried it myself, similarly to Google I presume you need to setup credentials for RDS connections. My understanding is that like GCP, it uses a Platform (GCP/AWS) with the right permissions to create that connection tunnel.

Reply:
AWS does not have a mandatory proxy like GCP, so with public accessible instance this example work. Of course most user will probably need to use a tunnel but I'm not sure if Gocloud could help for that.